src/main/resources/openapi/usermanagement.yaml

```diff
@@ -356,6 +364,11 @@ components:
     RoleDisplayState:
       type: boolean
 
+    RoleIds:
+      type: array
+      items:
+        type: integer
```

Comment:
or '#/components/schemas/RoleId' ?

Reply:
An alternative approach is to define an array type in-line within the `parameters.schema` directive, and `$ref` the `RoleId` type:

```yaml
  /admin/security-groups:
    get:
      summary: Get security groups
      parameters:
        - in: query
          name: role_ids
          description: Filter by list of role ids
          style: form
          explode: false
          schema:
            type: array
            items:
              $ref: '#/components/schemas/RoleId'
```

The general benefit is we don't clutter the `components.schema` namespace with lots of `FooList` types, and we get to reuse the existing `RoleId` definition.